tutorials/2d/using_tilesets.rst

```diff
@@ -605,6 +605,13 @@ atlas), but configured in different ways. For example, you may want to use the
 same tile image, but rotated, flipped, or modulated with a different color. This
 can be done using *alternative tiles*.
 
+.. tip::
+
+      Since Godot 4.2, you don't *have* to create alternative tiles to rotate or
```

Comment:
So even though it's not required is it recommended? I just ask because of the emphasis on *have* and if so that's  not clear

Reply:
It's neither required nor recommended. It's just a case of different use cases :slightly_smiling_face:

Reply:
I'd remove the * from "have" then. It just seems weird to have emphasis on it like this

Reply:
@Calinou I agree we could remove the emphasis here, other than that, this PR LGTM.